src/style.rs

```diff
@@ -261,13 +283,24 @@ impl ProgressStyle {
                 "elapsed_precise" => format!("{}", FormattedDuration(state.started.elapsed())),
                 "elapsed" => format!("{:#}", HumanDuration(state.started.elapsed())),
                 "per_sec" => format!("{}/s", state.per_sec()),
-                "bytes_per_sec" => format!("{}/s", HumanBytes(state.per_sec())),
-                "binary_bytes_per_sec" => format!("{}/s", BinaryBytes(state.per_sec())),
+                "bytes_per_sec" => format!("{}/s", HumanBytes(state.per_sec() as u64)),
+                "binary_bytes_per_sec" => format!("{}/s", BinaryBytes(state.per_sec() as u64)),
                 "eta_precise" => format!("{}", FormattedDuration(state.eta())),
                 "eta" => format!("{:#}", HumanDuration(state.eta())),
                 "duration_precise" => format!("{}", FormattedDuration(state.duration())),
                 "duration" => format!("{:#}", HumanDuration(state.duration())),
-                _ => "".into(),
+                key => {
```

Comment:
Put together like this, the separate `format_map.0.is_empty()` case looks silly. Let's assume that `.get()` will be reasonably quick in the case of a zero-length map.

I propose that we do `if let Some(_) = self.format.map.0.get(var)` before the `match var.key`, so that we can override the default keys.

Reply:
Seems reasonable. Hash table lookups, especially on small strings, should be quite fast.